internal/attestation/measurements/measurements.go

```diff
@@ -18,45 +18,76 @@ import (
 	"net/url"
 
 	"github.com/edgelesssys/constellation/v2/internal/attestation/vtpm"
+	"github.com/edgelesssys/constellation/v2/internal/cloud/cloudprovider"
 	"github.com/edgelesssys/constellation/v2/internal/sigstore"
 	"gopkg.in/yaml.v2"
 )
 
 // Measurements are Platform Configuration Register (PCR) values.
 type Measurements map[uint32][]byte
 
+// Zero returns a PCR value with all bits set to zero.
+func Zero() []byte {
+	return AllBytes(0x00)
+}
+
+// One returns a PCR value with all bits set to one.
+func One() []byte {
+	return AllBytes(0xFF)
+}
+
+// AllBytes returns a PCR value with all bytes set to b.
+func AllBytes(b byte) []byte {
+	return bytes.Repeat([]byte{b}, 32)
+}
+
+// DefaultsFor provides the default measurements for given cloud provider.
+func DefaultsFor(provider cloudprovider.Provider) Measurements {
+	switch provider {
+	case cloudprovider.AWS:
+		return awsPCRs
+	case cloudprovider.Azure:
+		return azurePCRs
+	case cloudprovider.GCP:
+		return gcpPCRs
+	case cloudprovider.QEMU:
+		return qemuPCRs
+	default:
+		return nil
+	}
+}
+
 var (
```

Comment:
You could use functions instead of global vars to make this code safer. Global state is never a good idea...

Reply:
Good hint! I moved them into `DefaultsFor`. It was the only reference anyway.